serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,274 @@
+2022-05-25 14:38:13,379 INFO  [org.kie.kog.qua.pro.dev.DataIndexInMemoryContainer] (docker-java-stream--938264210) STDOUT: 2022-05-25 17:38:13,182 INFO  [io.quarkus] (main) Profile prod activated.
+2022-05-25 14:38:13,380 INFO  [org.kie.kog.qua.pro.dev.DataIndexInMemoryContainer] (docker-java-stream--938264210) STDOUT: 2022-05-25 17:38:13,182 INFO  [io.quarkus] (main) Installed features: [agroal, cdi, hibernate-orm, hibernate-orm-panache, inmemory-postgres, jdbc-postgresql, narayana-jta, oidc, reactive-routes, rest-client-reactive, rest-client-reactive-jackson, security, smallrye-context-propagation, smallrye-graphql-client, smallrye-health, smallrye-metrics, smallrye-reactive-messaging, smallrye-reactive-messaging-http, vertx, vertx-graphql]
+----
+
+Once started, you can request the provided endpoint:
+
+[source,shell]
+----
+curl -X POST -H 'Content-Type:application/json' http://localhost:8080/hello_world
+----
+
+The response should be similar to:
+
+[source,shell]
+----
+{"id":"efb59bfa-ad9c-4062-a6d2-2d9184dd4b3d","workflowdata":{"greeting":"Hello World","mantra":"Serverless Workflow is awesome!"}}
+----
+
+Hit `CTRL+C` to stop the application, or keep it running and enjoy the blazing fast hot-reload.
+
+== Testing
+
+Now, let's add a test to make sure everything works as expected. But first, remove the
+auto-generated tests `org.kie.kogito.examples.GreetingResourceTest` and
+`org.kie.kogito.examples.NativeGreetingResourceIT`.
+
+Then, create the `org.kie.kogito.examples.HelloWorldTest` test class with the following content:
+
+[source,java]
+----
+package org.kie.kogito.examples;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+@QuarkusIntegrationTest
+class HelloWorldTest {
+
+    @BeforeAll
+    static void init() {
+        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
+    }
+
+    @Test
+    void testHelloEndpoint() {
+        given()
+                .contentType(ContentType.JSON)
+                .accept(ContentType.JSON)
+                .when().post("/hello_world")
+                .then()
+                .statusCode(201)
+                .body("workflowdata.greeting", is("Hello World"))
+                .body("workflowdata.mantra", is("Serverless Workflow is awesome!"));
+    }
+}
+----
+
+Before running the test, run the following command to build the application:
+[source,shell]
+----
+./mvnw package -DskipTests
+----
+
+Now you're ready to run the test:
+
+[source,shell]
+----
+./mvnw verify
+----
+
+You can also run the test from your IDE directly (be sure you stopped the application first).
+
+== What's next?
```

Comment:
People can also go to the Cloud category from here to understand how to deploy it in the cloud. cc @amadhusu